src/propagators/text_map_codec.js

```diff
@@ -93,7 +90,7 @@ export default class TextMapCodec {
         return spanContext;
     }
 
-    inject(spanContext: SpanContext, carrier: any): void {
+    inject(spanContext: SpanContext, carrier: any = {}): void {
```

Comment:
remove `= {}` defaulting. Calling this method with a carrier is a programming error.